packages/devtools-reps/src/reps/date-time.js

```diff
@@ -30,7 +30,7 @@ function DateTime(props) {
       getTitle(grip),
       span(
         { className: "Date" },
-        new Date(grip.preview.timestamp).toISOString()
+        new Date(grip.preview.timestamp).toLocaleDateString()
```

Comment:
I think we can safely use `toString`, which will resolve the issue we have :)

```suggestion
        new Date(grip.preview.timestamp).toString()
```

Reply:
Alright on it.

Reply:
and @nchevobbe I find this a little confusing, now that the tests read my current timezone, will that be a problem if the tests are run from any other time-zones?

Reply:
I actually just realized the pipeline fails since I'm testing using my time zone